spec/models/pseud_spec.rb

```diff
@@ -16,4 +16,42 @@
 
   # TODO: add more tests
 
-end
\ No newline at end of file
+  describe "save" do
+
+		before do
+			@user = User.new
+      @user.login = "mynamepseud"
+      @user.age_over_13 = "1"
+      @user.terms_of_service = "1"
+      @user.email = "foo1@archiveofourown.org"
+      @user.password = "password"
+      @user.save
```

Comment:
The tabs are not appearing in Github either now so I guess we'll have to chalk it up to a glitch somewhere